cmd/notification-service/wire.go

```diff
@@ -30,6 +31,7 @@ type Application struct {
 	EventPublisher     eventbus.Publisher
 	EntClient          *db.Client
 	FeatureConnector   feature.FeatureConnector
+	BillingService     billing.Service
```

Comment:
I am not sure if this is required.

Reply:
Agrred. I nthe previous version I used it for fetching hte invoice but that is not needed as the einvoce is available in the event itself. Will remove it.